kustomize/escalation/overlays/prod/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../../base
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+secretGenerator:
+- envs:
+  - secret.properties
+  name: escalation-secret
+configMapGenerator:
+- envs:
+  - config.properties
+  name: escalation-config
+patches:
+- path: sonataflow-patch.yaml
+  target:
+    kind: SonataFlow
+    name: ticketescalation
+
+images:
+- name: serverless-workflow-escalation
+  newName: quay.io/orchestrator/serverless-workflow-escalation
+  newTag: abcdef
```

Comment:
Where will quay.io/orchestrator/serverless-workflow-escalation@abcdef be obtained from? Will it be part of the CI pipeline?

Trying to verify we have a way to maintain compatibility between the soantaflow CR spec to the content of the specified image.

Reply:
`abcdef` is the default value that we commit but the first PR from the CI repo will replace it using the mutating command executed by the pipeline, like:
```
kustomize edit set image serverless-workflow-${WORKFLOW_ID}=${IMAGE_NAME}:${IMAGE_TAG}
```

This will result in a change similar to what you can see here:
```
images:
- name: serverless-workflow-escalation
  newName: quay.io/orchestrator/serverless-workflow-escalation
  newTag: 9a320174
```

Reply:
About the other comment

> Trying to verify we have a way to maintain compatibility between the soantaflow CR spec to the content of the specified image

I think the answer is ~yes: the CI pipelines guarantees to update the CR spec and the CR image at the same time, so they are consistent with the given commit SHA. Anyway, nothing prevents any developer to submit an inconsistent change to the manifest. In this case, the consistency can only be guaranteed by the operator at reconciliation time (anyway, IMO we cannot prevent the deployer from deploying a broken application, do we really want to achieve this level of control?)

Reply:
I don't think we need to. I was more concerned about the resources generated by our pipeline.